module/dice.js

```diff
@@ -374,12 +374,12 @@ async function getRollFormula(actor) {
       }
     }
 
-    if (rollData.environment.hasAuraBonus) {
+    if (rollData.environment.aura > 0) {
```

Comment:
Here it is more:
if (rollData.environment.aura && rollData.environment.aura.modifier != 0)

1. Aura is not always defined/used
2. Modifier can be negative